src/app/screens/sendStx/index.tsx

```diff
@@ -1,162 +1,143 @@
+  );
+  const [sendMax, setSendMax] = useState(false);
+  const [feeRate, setFeeRate] = useState(
+    previousFee ? microstacksToStx(new BigNumber(previousFee)).toString() : '',
+  );
+  const [unsignedSendStxTx, setUnsignedSendStxTx] = useState('');
 
-    if (memo) {
-      if (Buffer.from(memo).byteLength >= 34) {
-        setMemoError(t('ERRORS.MEMO_LENGTH'));
-        return false;
-      }
+  const handleCancel = () => {
+    if (isInOption) {
+      window.close();
+      return;
     }
-    return true;
-  }
+    navigate('/');
+  };
 
-  const onPressSendSTX = async (associatedAddress: string, amount: string, memo?: string) => {
-    const modifyAmount = replaceCommaByDot(amount);
-    const addMemo = memo ?? '';
-    if (validateFields(associatedAddress.trim(), modifyAmount, memo!)) {
-      setAddressError('');
-      setMemoError('');
-      setAmountError('');
-      mutate({ amount, associatedAddress, memo: addMemo });
+  const handleBackButtonClick = () => {
+    if (currentStep > 0) {
+      setCurrentStep(getPreviousStep(currentStep, true));
+    } else {
+      handleCancel();
     }
   };
 
-  return (
-    <>
-      <TopRow title={t('SEND')} onClick={handleBackButtonClick} />
-      <SendForm
-        processing={isLoading}
-        currencyType="STX"
-        amountError={amountError}
-        recepientError={addressError}
-        memoError={memoError}
-        balance={Number(microstacksToStx(new BigNumber(stxAvailableBalance)))}
-        onPressSend={onPressSendSTX}
-        recipient={recipientAddress!}
-        amountToSend={amountToSend!}
-        stxMemo={stxMemo!}
-      />
-      <BottomBar tab="dashboard" />
-    </>
+  const header = (
+    <TitleContainer>
+      <TokenImage token="STX" />
+      <Title>{t('SEND.SEND')}</Title>
+    </TitleContainer>
   );
+
+  switch (currentStep) {
+    case Step.SelectRecipient:
+      return (
+        <SendLayout selectedBottomTab="dashboard" onClickBack={handleBackButtonClick}>
+          <Container>
+            <Step1SelectRecipientAndMemo
+              header={header}
+              recipientAddress={recipientAddress}
+              setRecipientAddress={setRecipientAddress}
+              setRecipientDomain={setRecipientDomain}
+              memo={memo}
+              setMemo={setMemo}
+              onNext={() => setCurrentStep(getNextStep(Step.SelectRecipient, true))}
+              isLoading={isLoading}
+            />
+          </Container>
+        </SendLayout>
+      );
+    case Step.SelectAmount:
+      return (
+        <SendLayout selectedBottomTab="dashboard" onClickBack={handleBackButtonClick}>
+          <Container>
+            <Step2SelectAmount
+              header={header}
+              amount={amount}
+              setAmount={setAmount}
+              feeRate={feeRate}
+              setFeeRate={setFeeRate}
+              sendMax={sendMax}
+              setSendMax={setSendMax}
+              fee={feeRate}
```

Comment:
why do we need the `fee` prop when we already have `feeRate`? and we are passing the same value 🤔